examples/config-cve.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "0.1.0-dev",
+    "storage": {
+        "rootDirectory": "/tmp/zot"
+    },
+    "http": {
+        "address": "127.0.0.1",
+        "port": "8080"
+    },
+    "log": {
+        "level": "debug"
+    },
+    "extensions": {
+        "search": {
+            "cve": {
+                "periodicupdates": true
```

Comment:
Where in the code is this being used?

Reply:
This example field is not correct. Replace with UpdateInterval and test pls.

Reply:
Yes, I forget to update this file with latest changes that I had done regarding the CveConfig. I will replace with UpdateInterval.

I have added this config-cve file if we want to start the zot with cve extension also.